app/reporting.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Reporting
+analytics_key: reporting
+parent:
+  title: How to use the pilot service
+  url: /guide
+previous:
+  title: Recording a vaccination
+  url: /recording-a-vaccination
+next:
+  title: Managing users
+  url: /managing-users
+---
+
+{% include 'guide-contents.njk' %}
+
+Vaccination data is available for extract as soon as it’s recorded and saved. You can extract data in a ‘.csv format’ (comma separated values) to support reporting within your organisation by choosing from a combination of:
+
+* dates (up to a maximum of 2 weeks)
+* vaccines
+* sites
+
+You aren’t restricted to how often or when you can create reports.
+
```

Comment:
Should we say who is able to do this?

```suggestion

Only lead administrators and administrators can create reports. 

```

Reply:
Good shout, this should be right at the top like the managing user pages - so recorders can skip this page if they wish to.

Reply:
Done, thanks.